### ISSUE TITLE
RTX 3060 TI Dual mining

### ISSUE THREAD

**Issue by SkiffSalvatore** [trexminer/T-Rex#1092]
Is it possible to dual mine with a 3060 TI non LHR?

I've tried eth + rvn but rvn doesn't get any mh/s.

when I'm solo mining eth I get around 63 mh/s and with raven a steady 30 mh/s.

**Comment by SkiffSalvatore** [comment]
thanks for the replay. does this apply to non-lhr gpu's as well ?

**Comment by trexminer** [comment]
Yes, it's a workaround. eth+rvn dual mining hasn't been designed with non-LHR cards in mind.
